Week2/homework/js-exercises/whoWantDrink.js

```diff
@@ -0,0 +1,17 @@
+'use script'; 
+{
+    let drinkTray = [];
+    const drinkTypes = ['cola', 'lemonade', 'water'];
+    for (i = 0; i < 5; i++) {
+        if (i < 3) {
+        drinkTray.push(drinkTypes[i]);
+        }
+        else if (i < 4) {
+        drinkTray.push(drinkTypes[0]);
+        }
+        else {
+        drinkTray.push(drinkTypes[1]);
+        }
```

Comment:
Hehe, this works, but not the most scalable/reusable code. In JS2 you will learn about `filter()` which works well for this problem. Perhaps come back and think about how you can use it in this problem after JS ;)

Reply:
I thought I'd get this answer :) Yeah I will check it